well-sensor-esp32.ino

```diff
@@ -41,16 +45,7 @@ void setup() {
   initButton();
   initQr();
  
-  if (initSPIFSS()) {
```

Comment:
we need to think about how can we mock configurations
these addresses should be f part of device configurations and should be saved in spifss for sure - this may help to solve the problem
but that's another part - should not be taken into account right now